### PR TITLE
feat: add llm

### PR DESCRIPTION
**Pull request by mizy**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

### CODE REVIEW

**Review by huaxiabuluo — APPROVED** [review]
LGTM

File: server/api/studio/pkg/llm/llm_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"testing"
+
+	db "github.com/vesoft-inc/nebula-studio/server/api/studio/internal/model"
+)
+
+func TestImportJob_ReadPDFFile(t *testing.T) {
+	// Create a mock ImportJob instance
+	job := &ImportJob{
+		LLMJob: &db.LLMJob{
+			File: "data/llm/test.pdf",
```
Comment (veezhang) [inline server/api/studio/pkg/llm/llm_test.go:15]: It's exists?

Comment (veezhang) [reply server/api/studio/pkg/llm/llm_test.go:15]: Test files are usually placed in the `testdata` directory.

File: server/api/studio/pkg/llm/job.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		return job
+	}
+	return nil
+}
+
+func GetPendingJobs() []*db.LLMJob {
+	var jobs []*db.LLMJob
+	err := db.CtxDB.Where("status = ?", db.LLMStatusPending).Find(&jobs).Error
+	if err != nil && err != gorm.ErrRecordNotFound {
+		logx.Errorf("failed to get pending jobs: %v", err)
```
Comment (veezhang) [inline server/api/studio/pkg/llm/job.go:73]: Why ignore the `err`?

File: server/api/studio/pkg/llm/importjob.go [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+	defer file.Close()
+	_, err = file.WriteString(gqlStr)
+	if err != nil {
+		return nil, err
+	}
+	i.WriteLogFile(fmt.Sprintf("make gql file success, gqls length: %d", len(gqls)), "info")
+	return gqls, nil
+}
+
+func (i *ImportJob) RunGQLFile(gqls []string) error {
```
Comment (veezhang) [inline server/api/studio/pkg/llm/importjob.go:507]: never error?

**Pull request merged**